content/en/docs/v3.5/learning/why.md

```diff
@@ -118,13 +118,13 @@ Note that in the case of etcd keys, it can be locked efficiently because of the
 [etcd-linread]: ../api_guarantees/#isolation-level-and-consistency-of-replicas
 [etcd-mvcc]: ../data_model/
 [etcd-rbac]: ../../op-guide/authentication/#working-with-roles
-[etcd-recipe]: https://godoc.org/github.com/etcd-io/etcd/client/v3/experimental/recipes
+[etcd-recipe]: https://github.com/etcd-io/etcd/tree/main/client/v3/experimental/recipes
 [etcd-reconfig]: ../../op-guide/runtime-configuration
 [etcd-txn]: ../api/#transaction
-[etcd-v3election]: https://godoc.org/github.com/etcd-io/etcd/server/etcdserver/api/v3election/v3electionpb
-[etcd-v3lock]: https://godoc.org/github.com/etcd-io/etcd/server/etcdserver/api/v3lock/v3lockpb
+[etcd-v3election]: https://github.com/etcd-io/etcd/tree/main/server/etcdserver/api/v3election/v3electionpb
+[etcd-v3lock]: https://github.com/etcd-io/etcd/tree/main/server/etcdserver/api/v3lock/v3lockpb
```

Comment:
All `godoc.org` links should be replaced by corresponding https://pkg.go.dev/go.etcd.io links, not links into GH.

For example, this one is probably https://pkg.go.dev/go.etcd.io/etcd/server/v3/etcdserver/api/v3lock/v3lockpb
@nate-double-u @spzala, what do you think